Channels/Integration/01e_Magento2.md

```diff
@@ -0,0 +1,167 @@
+# Manage the Magento 2 connection
+
+To establish a connection to a Magento 2 shop, there are several particularities to consider. Therefore, the configuration of the Magento 2 connection is described in detail below.
+
+## Configure the Magento 2 connection   
+
+After the connection to a Magento 2 shop has been established, further settings can be configured for the connection.
+
+#### Prerequisites
+
+- A Magento 2 account has been created.
+- The *Magento 2* plugin has been installed.
+- A Magento 2 connection has been established.
+
+> [Info] For the *Omni-Channel* module version 4.1.0 or higher, the *Magento 2* plugin is required in at least version 1.0.0.
+
+[comment]: <> (Stimmt das so? In Platform Manager Magento 2 1.1.1 Version und Magento 2 B.B. 1.0.0 verfügbar.)
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab CONNECTIONS*
+
+![Magento 2 connection](../../Assets/Screenshots/Channels/Settings/Connections/Magento2/Connection.png "[Magento 2 connection]")
+
+1. Click the Magento 2 connection in the list of connections.   
+  The *Edit connection* view is displayed. By default, the *Credentials* tab is displayed.
+
+  ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Magento2/EditConnectionCredentials.png "[Edit connection credentials]")
+
+2. Click the *Settings* tab.   
+  The *Settings* tab is displayed. By default, the *Store view languages* setting is displayed. All store views configured in the webshop are displayed in the right side bar.
+
+  ![Edit connection settings](../../Assets/Screenshots/Channels/Settings/Connections/Magento2/EditConnectionSettings.png "[Edit connection settings]")
+
+3. Click the desired *Store view* drop-down list and select the appropriate language. The available languages are displayed in the drop-down list.  
```

Comment:
```suggestion
3. Click the *Store view* drop-down list and select the language in which the products will be uploaded to the store. All available languages are displayed in the list.  
```
Sofern du genauer spezifizieren kannst, was hier ausgewählt werden soll, solltest du das auch direkt hier beschreiben und nicht erst in einer Info, die eventuell niemand liest.
vgl. auch default sentences.